src/xbot/common/subsystems/BaseConnectionMonitorSubsystem.java

```diff
@@ -0,0 +1,40 @@
+package xbot.common.subsystems;
+
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.logic.Latch;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+import java.util.Observable;
+import java.util.Observer;
+
+public abstract class BaseConnectionMonitorSubsystem extends BaseSubsystem implements Observer {
```

Comment:
class probably doesn't need to be abstract if you don't have any abstract methods